docs/source/plotting/index.rst

```diff
@@ -0,0 +1,563 @@
+   text(-0.3, 0.45, "Figure")
+
+   color = 'blue'
+   ax.annotate('Spines', xy=(4.0, 0.35), xycoords='data',
+               xytext=(3.3, 0.5), textcoords='data',
+               weight='bold', color=color,
+               arrowprops=dict(arrowstyle='->',
+                               connectionstyle="arc3",
+                               color=color))
+
+   ax.annotate('', xy=(3.15, 0.0), xycoords='data',
+               xytext=(3.45, 0.45), textcoords='data',
+               weight='bold', color=color,
+               arrowprops=dict(arrowstyle='->',
+                               connectionstyle="arc3",
+                               color=color))
+
+   ax.text(4.0, -0.4, "Made with http://matplotlib.org",
+           fontsize=10, ha="right", color='.5')
+
+   #fig.show()
+
+
+Here are some of the highlights:
+
+* **Figure** is the main container in matplotlib. You can think of it as the page
+* **Axes** is the coordinate system. It contains most of the figure elements, such as Axis, Line2D, Text. 
+  One can have multiple Axes objects in one Figure
+* **Axis** is the container for the ticks and labels for the x and y axis of the plot
+
+======================
+Showing/saving figures
+======================
+
+There are two main ways that one can visualize images produced by matplotlib. The first one
+is to pop up a window with the required graph. For that, we use the `show()` function of the figure.
+
+.. code-block:: python
+
+   import matplotlib.pyplot as plt
+   fig,ax=plt.subplots()
+   #some code to generate figure
+   fig.show()
+
+If one wants to save the output, the figure object has a function called `savefig`.
+The main argument of savefig is the filename. Matplotlib will figure out the format of the figure
+from the file extension. The 'png', 'ps', 'eps', and 'pdf' extensions will work with
+almost any backend. For more information, see the documentation of
+`Figure.savefig <https://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure.savefig>`_
+Just replace the code above with:
+
+.. code-block:: python
+
+   import matplotlib.pyplot as plt
+   fig,ax=plt.subplots()
+   #some code to generate figure
+   fig.savefig('plot1.png')
+   fig.savefig('plot1.eps')
+
+
+Sometimes one wants to save a multi-page pdf document. Here is how to do this:
+
+.. code-block:: python
+
+   import matplotlib.pyplot as plt
+   from matplotlib.backends.backend_pdf import PdfPages
+
+   with PdfPages('/home/andrei/Desktop/multipage_pdf.pdf') as pdf:
+       #page1
+       fig,ax=plt.subplots()
+       ax.set_title('Page1')
+       pdf.savefig(fig)
+       #page2
+       fig,ax=plt.subplots()
+       ax.set_title('Page2')
+       pdf.savefig(fig)
+
+
+============
+Simple plots
+============
+
+For matrix workspaces, if we use the `mantid` projection, one can plot the data in a similar
+fashion as the plotting of arrays in matplotlib. Moreover, one can combine the two in the same figure
+
+.. plot::
```

Comment:
When plotting this in the workbench, my figure does not match the one produced here:
![image](https://user-images.githubusercontent.com/13132128/36666696-57acf992-1ae3-11e8-81c9-8b18406172eb.png)

Reply:
That's because you did not include the first import `from __future__ import division`. You can replace `20` with `20.`

Reply:
Ah good spot, this one is fine now. Thanks